variables.tf

```diff
@@ -232,3 +265,224 @@ variable "tags" {
+    })), [])
+    portal = optional(list(object({
+      host_name                       = string
+      key_vault_id                    = optional(string, null)
+      certificate                     = optional(string, null)
+      certificate_password            = optional(string, null)
+      negotiate_client_certificate    = optional(bool, false)
+      ssl_keyvault_identity_client_id = optional(string, null)
+    })), [])
+    developer_portal = optional(list(object({
+      host_name                       = string
+      key_vault_id                    = optional(string, null)
+      certificate                     = optional(string, null)
+      certificate_password            = optional(string, null)
+      negotiate_client_certificate    = optional(bool, false)
+      ssl_keyvault_identity_client_id = optional(string, null)
+    })), [])
+    proxy = optional(list(object({
+      host_name                       = string
+      default_ssl_binding             = optional(bool, false)
+      key_vault_id                    = optional(string, null)
+      certificate                     = optional(string, null)
+      certificate_password            = optional(string, null)
+      negotiate_client_certificate    = optional(bool, false)
+      ssl_keyvault_identity_client_id = optional(string, null)
+    })), [])
+    scm = optional(list(object({
+      host_name                       = string
+      key_vault_id                    = optional(string, null)
+      certificate                     = optional(string, null)
+      certificate_password            = optional(string, null)
+      negotiate_client_certificate    = optional(bool, false)
+      ssl_keyvault_identity_client_id = optional(string, null)
+    })), [])
+  })
+  default     = null
+  description = "Hostname configuration for the API Management service."
+}
+
+variable "min_api_version" {
+  type        = string
+  default     = null
+  description = "The version which the control plane API calls to API Management service are limited with version equal to or newer than."
+}
+
+variable "notification_sender_email" {
+  type        = string
+  default     = null
+  description = "Email address from which the notification will be sent."
+}
+
+variable "protocols" {
+  type = object({
+    enable_http2 = optional(bool, false)
+  })
+  default     = null
+  description = "Protocol settings for the API Management service."
+}
+
+variable "public_ip_address_id" {
+  type        = string
+  default     = null
+  description = "ID of a standard SKU IPv4 Public IP. Only supported on Premium and Developer tiers when deployed in a virtual network."
+}
+
+variable "public_network_access_enabled" {
+  type        = bool
+  default     = true
+  description = "Is public access to the API Management service allowed? This only applies to the Management plane, not the API gateway or Developer portal."
+  nullable    = false
+  # validation {
```

Comment:
is there a reason this validation is commented out? does it not work, or is it no longer needed?

Reply:
do not remember why I added that validation in the first place or why I commented it out, removed it now